genshi/filters/html.py

```diff
@@ -354,7 +354,7 @@ def __init__(self, safe_tags=SAFE_TAGS, safe_attrs=SAFE_ATTRS,
     # IE6 <http://openmya.hacker.jp/hasegawa/security/expression.txt>
     #     7) Particular bit of Unicode characters
     _URL_FINDITER = re.compile(
-        u'[Uu][Rr\u0280][Ll\u029F]\s*\(([^)]+)').finditer
+        u'[Uu][Rr\u0280][Ll\u029F]%s*\(([^)]+)' % (r'\s')).finditer
```

Comment:
This is the only way I could easily keep the unicode string for python2 support.

Reply:
A bit ugly, but it works!